docs/_static/site.css

```diff
@@ -0,0 +1,38 @@
+
+.iver-logo-left, .iver-logo-right {
+	line-height: 100px;
```

Comment:
Should we use px or rem? just thinking since we brought this up on another MR.

Reply:
Oh snap, caught red handed.

Yes, yes we should use `rem`. I'll fix

Reply:
REM'ed it up